simulariumio/constants.py

```diff
@@ -170,3 +170,8 @@ def JMOL_COLORS() -> pd.DataFrame:
 ]
 
 MAX_AGENT_ID = 0x80000000  # Agent IDs should fit in a 32 bit signed int
+
+
+class VIEWER_DIMENSION_RANGE:
+    MIN = 5
+    MAX = 70
```

Comment:
as a software engineer I would be more comfortable using "4" (or "8") and "64" but it probably makes no difference.  I wonder what kind of scene would break this?  A very very densely packed one , I guess?